libs/ml/include/ml/model/model.hpp

```diff
@@ -17,19 +17,12 @@
 //
 //------------------------------------------------------------------------------
 
-#include "ml/core/graph.hpp"
-#include "ml/dataloaders/tensor_dataloader.hpp"
-#include "ml/meta/ml_type_traits.hpp"
 #include "ml/model/model_config.hpp"
 #include "ml/ops/loss_functions/types.hpp"
```

Comment:
anticipating questions, 
1) model config is used during model construction:
 explicit Model(ModelConfig<DataType> model_config = ModelConfig<DataType>())
2) a forward declaration isn't applicable here because of such function declaration:
void Compile(OptimiserType optimiser_type, ops::LossType loss_type = ops::LossType::NONE...

Reply:
👍